src/lumigo_tracer/utils.py

```diff
@@ -17,6 +17,7 @@
 SECONDS_TO_TIMEOUT = 0.5
 LUMIGO_EVENT_KEY = "_lumigo"
 STEP_FUNCTION_UID_KEY = "step_function_uid"
+MIN_SPAN_SIZE = 300
```

Comment:
Why do you need the mean span size?

Reply:
I encounter the following scenario:
We entered 100 out of 400 available spans, and the current size was 9999, and the limit was 10000. 
And then we tried to find another span to fit this space although it is not possible, we count the sizes of all the other 300 spans and waste time for nothing...

Reply:
Why 300 and not 100? What's special about this number

Reply:
nothing special. It just sounds right:
lambda span = 1300 characters:
```
{'span_id': '946345f5-e331-4938-8be1-6cf5e3b8096b',
 'aws_account': '114300393969',
 'tracer_version': '1.1.92',
 'transaction_id': 'b25fdc368c4ce799550e856d',
 'message_ids': None,
 'span_type': 'function',
 'started': 1592188575734,
 'ended': 1592188591135,
 'version': 1,
 'token': 't_1b8e3e1eada1064d41ff',
 'raw': {},
 'region': 'us-west-2',
 'customer': 'c_ada1064d41ff',
 'aws_traceid_root': '1-0000b4c2-b25fdc368c4ce799550e856d',
 'resource_name': 'prod_resources-details-stls_collect-resources',
 'readiness': 'warm',
 'log_group_name': '/aws/lambda/prod_resources-details-stls_collect-resources',
 'log_stream_name': '2020/06/15/[$LATEST]8bec5ffb67ae4df489ffabebff464ff2',
 'error': None,
 'memory_allocated': '256',
 'runtime': 'AWS_Lambda_python3.7',
 'trigger_details': None,
 'event': {'data': {'customer_id': 'c_b932406854b74'}, 'isComplete': True},
 'envs': {'data': {'LUMIGO_AUTO_TAG_API_GW_HEADERS': 'customer_id'},'isComplete': False},
 'return_value': 'None',
 'max_finish_time': None,
 'warnings': [],
 'reporter_rtt': 81,
 'cost': 6.4635e-05,
 'created_from_logs': False,
 'invoked_version': None,
 'invoked_arn': None,
 'execution_tags': [],
 'delete_on': 1595471791,
 'db_insert_time': 1592188598717,
 'customer_arn': 'c_ada1064d41ff_arn:aws:lambda:us-west-2:114300393969:function:prod_resources-details-stls_collect-resources'}
```
HTTP span = 700 characters:
```
{
  'span_id': '58ca229c-829b-f5a7-85e4-41793e26debd',
  'aws_account': '563186419109',
  'tracer_version': '1.16.0',
  'transaction_id': 'aebafbefca67a705c3fbdde9',
  'message_ids': None,
  'span_type': 'http',
  'started': 1590017586012,
  'ended': 1590017586405,
  'version': 1,
  'token': 't_eda06e5420444c7aa73fb',
  'raw': {
    
  },
  'region': None,
  'customer': 'c_173b4d442b284',
  'aws_traceid_root': '1-5ec5be31-aebafbefca67a705c3fbdde9',
  'resource_name': 'localhost',
  'parent_id': '9312d926-032a-5ae0-9e63-8e19c35a79d2',
  'host': 'localhost',
  'request': {
    'body': '""'
  },
  'response': {
    'headers': {
      'status_code': 200
    }
  },
  'delete_on': 1593300786,
  'db_insert_time': 1590017603787
}
```

Reply:
nothing special. It just sounds right:
lambda span = 950 characters:
```
{
  'id': '91c8c328-f963-4bbf-9a5c-fee8d6a10d26_started',
  'type': 'function',
  'name': 'tracer-test-saart-main_python37',
  'runtime': 'AWS_Lambda_python3.7',
  'memoryAllocated': '256',
  'readiness': 'cold',
  'info': {
    'logStreamName': '2020/05/18/[$LATEST]ead51cda96374112aeb1c02f7720d8c8',
    'logGroupName': '/aws/lambda/tracer-test-saart-main_python37',
    'triggeredBy': 'unknown',
    'tracer': {
      'version': '1.1.86'
    },
    'traceId': {
      'Root': '1-5ec2569b-6bad899d8ef441958cf57ce9'
    }
  },
  'lumigo_execution_tags_no_scrub': [
    
  ],
  'started': 1589794459944,
  'transactionId': '6bad899d8ef441958cf57ce9',
  'account': '723663554526',
  'region': 'us-west-2',
  'parentId': '91c8c328-f963-4bbf-9a5c-fee8d6a10d26',
  'event': '{"functionName": "transactions/basic_invoke_lambda"}',
  'envs': '{"LUMIGO_DEBUG": "true"}',
  'ended': 1589794459944,
  'maxFinishTime': 1589794467943,
  'token': 't_b3db5c81ad0de65f9d99'
}
```
HTTP span = 850 characters:
```

  {
    'info': {
      'messageId': '5ebc595f-36f3-4b1d-b63d-1756ee170626',
      'httpInfo': {
        'host': 'lambda.us-west-2.amazonaws.com',
        'response': {'body': ''}
        'request': {'body': '',
        }
      },
      'tracer': {
        'version': '1.1.86'
      },
      'traceId': {
        'Root': '1-5ec2569b-6bad899d8ef441958cf57ce9'
      }
    },
    'id': '5ebc595f-36f3-4b1d-b63d-1756ee170626',
    'type': 'http',
    'ended': 1589794463195,
    'name': None,
    'invocationType': 'RequestResponse',
    'started': 1589794461205,
    'transactionId': '6bad899d8ef441958cf57ce9',
    'account': '723663554526',
    'region': 'us-west-2',
    'parentId': '91c8c328-f963-4bbf-9a5c-fee8d6a10d26',
    'event': '{"functionName": "transactions/basic_invoke_lambda"}',
    'envs': '{"LUMIGO_DEBUG": "true"},
    'token': 't_b3db5c81ad0de65f9d99'
  }
```